security/cmd/istio_ca/main.go

```diff
@@ -274,6 +280,23 @@ func runCA() {
 				Namespace:   opts.istioCaStorageNamespace,
 			}
 		}
+		if len(opts.customDNSNames) > 0 {
+			customNames := strings.Split(opts.customDNSNames, ",")
+			for _, customName := range customNames {
+				nameDomain := strings.Split(customName, ":")
+				if len(nameDomain) == 2 {
+					override, ok := webhooks[nameDomain[0]]
+					if ok {
+						override.CustomDomains = append(override.CustomDomains, nameDomain[1])
```

Comment:
The variable 'override' is not used.

Reply:
override.CustomDomains is modified: if we already have an entry, we add an additional domain to it.